scripts/keeper.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
if we keep this script here, we need to update the setup part in the tutorial, otherwise it's not clear why it is here for users. Otherwise we can simply remove it, copying contracts is handled in the tutorial in a more simple way https://docs.oceanprotocol.com/tutorials/react-setup/#copy-contract-artifacts

Reply:
agreed